DPRHTML/index.html

```diff
@@ -1,11 +1,18 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" manifest="dpr.appcache">
```

Comment:
does this not work well with pwa? 

removing this will heavily impact non-pwa scenarios. e.g. desktop.

Reply:
I thought precaching with workbox would replace appcache?

Reply:
Ok I put it back.